Carbon.IIS/Functions/Get-CIisHttpRedirect.ps1

```diff
@@ -37,16 +37,17 @@ function Get-CIisHttpRedirect
     param(
         # The site's whose HTTP redirect settings will be retrieved.
         [Parameter(Mandatory)]
-        [String] $SiteName,
+        [Alias('SiteName')]
+        [String] $LocationPath,
 
-        # The optional path to a sub-directory under `SiteName` whose settings to return.
-        [String] $VirtualPath = ''
+        # OBSOLETE. Use the `LocationPath` parameter instead.
+        [Alias('Path')]
+        [String] $VirtualPath
     )
 
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -Session $ExecutionContext.SessionState
 
-    Get-CIisConfigurationSection -SiteName $SiteName `
-                                 -VirtualPath $VirtualPath `
-                                 -SectionPath 'system.webServer/httpRedirect'
+    $sectionPath = 'system.webServer/httpRedirect'
+    Get-CIisConfigurationSection -LocationPath $LocationPath -VirtualPath $VirtualPath -SectionPath $sectionPath

```

Comment:
Both LocationPath and VirtualPath seem at odds in this call.

Reply:
Here (and in other places), I'm trying to maintain as much backwards-compatability as possible so users have at least one major version where they can remove obsolete functionality. Old way was to pass in a website name and the path to the application/virtual directory/directory. New way is to use a single parameter for that, `LocationPath` (which is just the site name joined with the virtual path).

Both the old `VirtualPath` and new `LocationPath` get passed to `Get-CIisConfigurationSection` because `Get-CIisConfigurationSection` handles if someone is still using `VirtualPath` by writing a warning. It is assumed if someone is using `VirtualPath`, they're most likely using `SiteName` as the first parameter name instead of `LocationPath`.